### PR TITLE
chore: reintroduce webRTC + webRTC direct interop tests

### PR DESCRIPTION
**Pull request by maschad**
## Title

chore: reintroduce webRTC and webRTC direct transport interop tests 
## Description

There has been a lot of fixes added to webRTC and so following the removal https://github.com/libp2p/js-libp2p/pull/1845 due to flakiness as mentioned here https://github.com/libp2p/js-libp2p/issues/1846 we want to see if these fixes have resolved the issues.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by achingbrain** [comment]
Tests will be fixed by https://github.com/libp2p/js-libp2p/pull/2234

File: interop/firefox-version.json [hunk 1]
```diff
@@ -1,7 +1,16 @@
 {
-  "id": "firefox-js-libp2p-head",
-  "containerImageID": "firefox-js-libp2p-head",
+  "id": "chromium-js-libp2p-head",
```
Comment (achingbrain) [inline interop/firefox-version.json:2]: Why is `chromium-js-libp2p-head` in `interop/firefox-version.json`?

File: interop/node-version.json [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-    "secureChannels": [
-        "noise"
-    ],
-    "muxers": [
-        "mplex",
-        "yamux"
-    ]
-}
\ No newline at end of file
+  "id": "firefox-js-libp2p-head",
```
Comment (achingbrain) [inline interop/node-version.json:2]: Why is `firefox-js-libp2p-head` in `interop/node-version.json`?

File: interop/node-version.json [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
+  "containerImageID": "firefox-js-libp2p-head",
+  "transports": [
+    {
+      "name": "webrtc-direct",
```
Comment (achingbrain) [inline interop/node-version.json:6]: node.js has no webrtc-direct support - is this a mistake?

Comment (maschad) [reply interop/node-version.json:6]: Definitely a mistake 😓

File: interop/firefox-version.json (hunk 1 above)
Comment (maschad) [reply interop/firefox-version.json:2]: copy pasta error 😓

File: interop/node-version.json (hunk 2 above)
Comment (maschad) [reply interop/node-version.json:2]: good catch, whoops

**Review by achingbrain — APPROVED** [review]

**Pull request merged**